From 0e5acd265bd9762203f3ca9dcd72e9c13971f0c3 Mon Sep 17 00:00:00 2001
From: Adrien Ricciardi <aricciardi@baylibre.com>
Date: Fri, 6 Oct 2023 11:38:04 +0200
Subject: [PATCH 28/38] examples: posix: Removed the now useless protocol
 version field of the CAN controllers configurations from the CAN demo.

Signed-off-by: Adrien Ricciardi <aricciardi@baylibre.com>
---
 examples/posix/can_demo/can_demo.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/examples/posix/can_demo/can_demo.c b/examples/posix/can_demo/can_demo.c
index 2b4369e6..e08d5beb 100644
--- a/examples/posix/can_demo/can_demo.c
+++ b/examples/posix/can_demo/can_demo.c
@@ -39,7 +39,6 @@ int main(void)
 		// First controller will use CAN 2.0 (values are fake)
 		{
 			&can_demo_driver_controller_1,
-			CAN_PROTOCOL_VERSION_CLASSIC,
 			{
 				.CanControllerBaudRate = 250,
 				.CanControllerBaudRateConfigID = 0,
@@ -53,7 +52,6 @@ int main(void)
 		// Second controller will use CAN FD with bit rate switch (values are fake)
 		{
 			&can_demo_driver_controller_2,
-			CAN_PROTOCOL_VERSION_FD,
 			{
 				.CanControllerBaudRate = 1000,
 				.CanControllerBaudRateConfigID = 1,
-- 
2.34.1

